frontend/src/components/ParkDetailsSubArea.jsx

```diff
@@ -0,0 +1,29 @@
+import ParkSeason from "@/components/ParkDetailsSeason";
+// @TODO: map icons to feature types
+import walkInCamping from "@/assets/icons/walk-in-camping.svg";
+import PropTypes from "prop-types";
+
+export default function SubArea({ title, data }) {
+  return (
+    <section className="sub-area">
+      <h2>
+        <img src={walkInCamping} className="sub-area-icon" /> {title}
+      </h2>
+
```

Comment:
Would be great if we could use API data to display icons.

Reply:
Agreed 👍 I think those values should come over with other metadata when we sync our data with Strapi, but right now we don't have a field for it.